source/includes/_custom_fields.md

```diff
@@ -23,6 +23,7 @@ window.wootricSettings = {
     pricing_plan:'Enterprise', // TODO: The current user's pricing plan.
     total_purchase_amount: 12, // Integer representing the user's total purchases with the key suffixed with "_amount"
     last_order_date: 1350466020 // Integer representing the date (Unix timestamp format) of the user's last order with the key suffixed with "_date"
+    items_array: ['ITEM1', 'ITEM2'] // List of values representing the possible items with the key suffixed with "_array"

```

Comment:
This shouldn't be an array, it's a comma separated string.

Reply:
OK, got it, thanks Diego

Reply:
> The values for `_array` properties should be strings. Example: `'Item1, Item2'`
> 
> I think they _could_ arrays but the web-beacon hasn't been update to handle that yet., so better to stick with comma separated strings.

Can we discuss when you return?

Reply:
ok updated!